plugins/ipam/dhcp/dhcp_test.go

```diff
@@ -107,6 +122,7 @@ var _ = BeforeSuite(func() {
 var _ = AfterSuite(func() {
 	os.Remove(socketPath)
 	os.Remove(pidfilePath)
+	defer os.RemoveAll(tmpDir)
```

Comment:
And actually, we don't even need this right?  We should put the RemoveAll(tmpDir) inside the AfterEach(), since it'll be unique to each testcase.  This RemoveAll() call in AfterSuite would only remove the last testcase tmpdir, which should already have been done by AfterEach()